src/ayab/knitter.cpp

```diff
@@ -390,6 +390,14 @@ bool Knitter::calculatePixelAndSolenoid() {
 
   bool beltShift = BeltShift_t::Shifted == m_beltShift;
 
+  // 270 Doesn't care about belt shift
+  if (Machine_t::Kh270 == m_machineType) {
+    beltShift = false;
+  }
+
+  // 270 needs additional start offsets because of it's wierdness
+  uint8_t bulkyOffset = 0;
```

Comment:
Instead of introducing yet another variable, you could remove it and modify directly startOffset below

Reply:
For me personally, the additional variable makes it a bit more transparent why magic numbers are added.